src/main/resources/application.yaml

```diff
@@ -51,7 +51,7 @@ spring:
 
 # setting env vars + defaults as env vars will be overridden otherwise
 reports:
-  cron: ${REPORTS_CRON:0 0 18 * * *}
+  cron: ${REPORTS_CRON:0 0 18 ? * W}
```

Comment:
>W (weekday) – it is used to specify the weekday (Monday to Friday) nearest to a given day of the month. For example, if we specify “10W” in the <day-of-month> field, then it means the “weekday near to 10th of that month”. So if “10th” is a Saturday, then the job will be triggered on “9th”, and if “10th” is a Sunday, then it will trigger on “11th”. If you specify “1W” in the <day-of-month> and if “1st” is Saturday, then the job will be triggered on “3rd” which is Monday, and it will not jump back to the previous month

`MON-FRI` can be used 

https://www.baeldung.com/cron-expressions

Reply:
I read it correctly, but then after doing all text still assumed as workday. derp